### ISSUE TITLE
When to update xgboost Training API

### ISSUE THREAD

**Issue by skyang2009** [nuanio/xgboost-node#11]
hi, is there any plan to update the xgboost training API in the near future, it would be very useful to train xgboost model with nodejs

**Comment by qinwf-nuan** [comment]
No, it is recommended to train model in python for now. Thanks for asking.

**Comment by tuckerconnelly** [comment]
Would you accept a PR? And could you point us in the right direction for adding a train function?

**Comment by qinwf-nuan** [comment]
Yes, we accept PRs.

Training function will require:

1. pandas like or matrix IO and processing functions
2. https://github.com/nuanio/xgboost-node/blob/master/xgboost/include/xgboost/c_api.h#L358 and https://github.com/nuanio/xgboost-node/blob/master/xgboost/include/xgboost/c_api.h#L345 to do training.
